### PR TITLE
Avoid changing border width or padding on hover

### PR DESCRIPTION
**Pull request by urkerab**
Cosmetic issue whereby if you're not at 100% zoom then moving pixels from padding to border causes layout to get recalculated. In my case it causes the wastebasket icon for the "Delete" button (but not the text!) to move when the edit button is hovered at 90% zoom.

I noticed the problem with the teamlist button but I changed the other hover too just in case. (I didn't find any other hovers which changed the border or padding.)

### CODE REVIEW

**Comment by Zarel** [comment]
Yeah, I do it this way by habit because IE7 doesn't support transparent borders, but I don't think we support any browsers that have trouble with transparent borders, so this approach is much better.

File: style/client.css [hunk 1]
```diff
@@ -2301,7 +2301,6 @@ a.ilink.yours {
 .setmenu button:hover,
 .teamlist button:hover {
 	border: 1px solid #888888;
```
Comment (Zarel) [inline style/client.css]: just do `border-color:`

File: style/client.css [hunk 2]
```diff
@@ -2724,20 +2723,19 @@ a.ilink.yours {
...
 	border-radius: 4px;
-	padding: 1px 1px 1px 5px;
+	padding: 0px 0px 0px 4px;
 	margin: 0 5px 1px 5px;
 	background: transparent;
 	cursor: pointer;
 }
 .teambuilder-results .result a:hover {
-	padding: 0px 0px 0px 4px;
 	border: 1px solid #D8D8D8;
```
Comment (Zarel) [inline style/client.css]: ditto

File: style/client.css [hunk 3]
```diff
@@ -2724,20 +2723,19 @@ a.ilink.yours {
...
 	background: #F8F8F8;
 }
 .teambuilder-results .firstresult a,
 .teambuilder-results .firstresult a:hover {
-	padding: 0px 0px 0px 4px;
 	border: 1px solid #CCCCCC;
```
Comment (Zarel) [inline style/client.css]: ditto

Comment (urkerab) [inline style/client.css]: Actually that one won't work because there's a non-hover rule.

**Pull request merged**